Jenkinsfile

```diff
@@ -0,0 +1,81 @@
+pipeline {
+  agent any
+  stages {
+    stage('build docker image and tag') {
+      steps {
+        sh 'sudo docker build -t ssoweb .'
+      }
+    }
+    stage('start env') {
+      when {
+                changeRequest()
+            }
+      steps {
+        sh '''export PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/root/bin
+        docker-compose up -d'''
+      }
+    }
+    stage('run unit test') {
+      when {
+                changeRequest()
+            }
+      steps {
+        sh './unit_test.sh'
+      }
+    }
+    stage('destroy env') {
+      when {
+                changeRequest()
+            }
+      steps {
+        sh '''export PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/root/bin
+        docker-compose down -v'''
+      }
+    }
+    stage('ecr push') {
+      when {
+                branch 'master'
+            }
+      steps {
+        sh '''eval sudo $(aws ecr get-login --no-include-email --region us-west-2)
+latest_tag=$(aws ecr describe-images --repository-name ssoweb --region us-west-2  --output text --query \'sort_by(imageDetails,& imagePushedAt)[*].imageTags[*]\' | tr \'\\t\' \'\\n\'  | tail -1)
+VERSION_TAG="${latest_tag%.*}.$((${latest_tag##*.}+1))"
+sudo docker tag ssoweb 958491237157.dkr.ecr.us-west-2.amazonaws.com/ssoweb:${VERSION_TAG}
```

Comment:
This is a public repository. Should this information be checked in here? Ditto for the next line.